### PR TITLE
adding cw-eval recipe

### PR DESCRIPTION
**Pull request by nrweir**
Recipe for the cw-eval package from PyPI.

### CODE REVIEW

**Comment by nrweir** [comment]
@conda-forge-admin I believe this is ready to be merged, let me know if more is needed on my end.

**Comment by nrweir** [comment]
@conda-forge/staged-recipes I believe this is ready to go!

File: recipes/cw-eval/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+
+test:
+  imports:
+    cw_eval
+
+about:
+  home: http://github.com/cosmiq/cw-eval
+  license: Apache-2.0
+  license_family: Apache
+  license_file: LICENSE.txt
```
[the remark this answers is not in the document]
Comment (nrweir) [reply recipes/cw-eval/meta.yaml]: @scopatz Thanks! apologies for the bad link. Should be fixed in the new commit.

[the remark this answers is not in the document]
Comment (scopatz) [reply recipes/cw-eval/meta.yaml]: No problem!

**Pull request merged**